### DISCUSSION TITLE
0.70-rc missing ?

### DISCUSSION THREAD

**Discussion by darth-hp** [HASwitchPlate/openHASP#760] [Q&A] [answered]
Hi guys,

I have a CYD v3 which has a ST7789 display. I wanted to compile the dev branch on my own and followed the instructions in https://www.openhasp.com/0.7.0/firmware/compiling/local/
Unfortunately the checkout fails because the 07.0-rc branches don't exist. Any idea?

**Comment by fvanroie** [comment, marked as answer]
0.7.0-rc _is_ the master branch.

**Reply by darth-hp** [reply]
Ah! Then maybe the document should be updated?

**Reply by fvanroie** [reply]
Click the Pencil button on the top of the page and post a PR, thanks.
